packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -109,13 +115,20 @@ function EditorCanvas( { enableResizing, settings, children, ...props } ) {
 			iframeProps={ {
 				expand: isZoomOutMode,
 				scale: isZoomOutMode ? 0.45 : undefined,
-				frameSize: isZoomOutMode ? 100 : undefined,
+				frameSize: isZoomOutMode ? 50 : undefined,
 				className: classnames(
 					'edit-site-visual-editor__editor-canvas',
 					{
 						'is-focused': isFocused && canvasMode === 'view',
 					}
 				),
+				maxHeight:
+					isZoomOutMode && containerHeight
```

Comment:
I guess we wouldn't want to set a maxHeight in `isZoomOutMode` always.

Right now I'm just testing in the site editor when browsing styles.